pkg/downwardmetrics/vhostmd/disk.go

```diff
@@ -102,7 +102,10 @@ func readDisk(filePath string) (*Disk, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer f.Close()
+	// It is safe to ignore error that may happen when closing a file, because it is only used for reading.
+	// If the read operation succeeds, but close fails, we have already read the data, so it is safe to ignore.
+	defer func() { _ = f.Close() }()
```

Comment:
We have `CloseIOAndCheckErr` that can be used.

Reply:
That's useful function, thanks.